build.gradle.kts

```diff
@@ -85,6 +90,14 @@ signing {
     useInMemoryPgpKeys(System.getenv("GPG_KEY"), System.getenv("GPG_KEY_PASSWORD"))
 }
 
+spotless {
+    kotlin {
+        ktlint(libs.ktlint.get().version)
+    }
+    kotlinGradle {
+        ktlint(libs.ktlint.get().version)
+    }
+}
```

Comment:
Is that going to fail the build everytime there is a code style violation? 

As much as I welcome `.editorconfig` (and even better, `.idea/codeStyle`), I have very mixed feelings about adding more friction to development. [This blog post explains why](https://dev.to/autonomousapps/acab-fire-the-code-style-cop-in-your-head-m8b)

Reply:
Yeah, `check` depends on `spotlessCheck`. 
`.editorconfig` would be enough, it's honored by IDEA, VSC, and others.

Reply:
LGTM 👍 Can you keep `.editorconfig` but remove spotless?

Reply:
I can. But some rules break the rules, e.g. https://github.com/GradleUp/nmcp/blob/350f1f3c250fa4ba27978de517d013aa8eb4f65b/.editorconfig#L11

Reply:
Not sure I'm following. Do you mean that some of the existing code breaks the rule? This is fine, we can reformat as we go. Or just commit the reformat alongside `.editorconfig`

Reply:
2c18fcf72915adb495a2f6c2c3e4342da38f886a